xml/System.Threading/Timer.xml

```diff
@@ -386,8 +386,8 @@ Sub TimerCallback(state As Object)
       <Docs>
         <param name="callback">A delegate representing a method to be executed.</param>
         <param name="state">An object containing information to be used by the callback method, or <see langword="null" />.</param>
-        <param name="dueTime">The amount of time to delay before the <paramref name="callback" /> parameter invokes its methods. Specify negative one (-1) milliseconds to prevent the timer from starting. Specify zero (0) to start the timer immediately.</param>
-        <param name="period">The time interval between invocations of the methods referenced by <paramref name="callback" />. Specify negative one (-1) milliseconds to disable periodic signaling.</param>
+        <param name="dueTime">The amount of time to delay before the <paramref name="callback" /> is invoked, <see cref="F:System.Threading.Timeout.InfiniteTimeSpan" /> to prevent the timer from starting. Specify <see cref="F:System.TimeSpan.Zero" /> to start the timer immediately.</param>
```

Comment:
We should keep the same structure used elsewhere, i.e. go back to using ". Specify" rather than combining the sentence.

Reply:
If you see anything else let me know, sorry for the waste of your time and thank you.